polaris-react/src/components/Frame/components/Toast/Toast.tsx

```diff
@@ -1,9 +1,5 @@
 import React, {useEffect} from 'react';
```

Comment:
To see toast, make a change and save

[This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-frame--in-an-application)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-frame--in-an-application&globals=polarisSummerEditions2023:true)